src/java.base/share/classes/sun/security/provider/SunEntries.java

```diff
@@ -103,6 +103,9 @@ public final class SunEntries {
      */
     private static final boolean useNativeDigest = NativeCrypto.isAlgorithmEnabled("jdk.nativeDigest", "MessageDigest");
 
+    // Flag indicating whether the operating system is AIX.
+    private static final boolean isAIX = "AIX".equals(GetPropertyAction.privilegedGetProperty("os.name"));
+
```

Comment:
Rather than add yet another O/S test, suggest `import jdk.internal.util.OperatingSystem;` and use `OperatingSystem.isAix()` below.

Reply:
Yes, we can try to find something equivalent for jdk17 and older versions when we get there.

Reply:
Changed to suggestion.